changelogs/api-management-service.md

```diff
@@ -1,5 +1,49 @@
 # Azure API Management service changelog
 
+Releases are deployed gradually in phases and batches, [following the safe deployment practices framework and release channels](https://learn.microsoft.com/azure/api-management/validate-service-updates). Rollout may take several weeks across Azure regions, so new features and fixes may not be immediately available in your service.
+
+## Release - API Management service: September, 2025
+
+### Highlights
+
+Since the last update, we’ve added:
```

Comment:
We should sanitize non-standard apostrophes throughout.

Reply:
Keen eye! Fixed with the next commit.